pkg/executables/config/kind.yaml

```diff
@@ -19,6 +19,25 @@ kubeadmConfigPatches:
         imageTag: {{.EtcdVersion}}
     imageRepository: {{.KubernetesRepository}}
     kubernetesVersion: {{.KubernetesVersion}}
+{{- if (ne .ECRMirrorEndpoint "") }}
+containerdConfigPatches:
+  - |
+    [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
+      [plugins."io.containerd.grpc.v1.cri".registry.mirrors."public.ecr.aws"]
+        endpoint = ["https://{{.ECRMirrorEndpoint}}"]
+      [plugins."io.containerd.grpc.v1.cri".registry.configs."{{.ECRMirrorEndpoint}}".tls]
+{{- if (eq .ECRMirrorCert "") }}
+        insecure_skip_verify = true
+{{- else }}
+        ca_file = "/etc/containerd/certs.d/{{.ECRMirrorEndpoint}}/ca.crt"
+nodes:
+- role: control-plane
+  extraMounts:
+    - containerPath: /etc/containerd/certs.d/{{.ECRMirrorEndpoint}}/ca.crt
+      hostPath: {{.ECRMirrorCert}}
```

Comment:
It looks like the cert is optional, should we only mount if they provide one?  Is it really a CA, `ECRMirrorCA` might be more clear?

Reply:
Yep the cert is optional. If users don't provide a cert, we will just disable TLS for pulling from registry. 
The mounting logic is under the `{{- else }}` block on line 31, so the cert will only be mounted if one is provided. 
I can update the field name to make it more clear.

Reply:
oh right.